internal/tiltfile/api/__init__.py

```diff
@@ -675,6 +676,7 @@ def local(command: Union[str, List[str]],
     echo_off: If set to True, skips printing command to log.
     env: Environment variables to pass to the executed ``command``. Values specified here will override any variables passed to the Tilt parent process.
     dir: Working directory for ``command``. Defaults to the Tiltfile's location.
+    stdin: If non-empty, will be written to ``command``'s stdin.
```

Comment:
Or maybe #3442 will want an empty stdin so this should be "if non-None"

Reply:
ya, we should handle None and empty string differently? i think there are some subtle differences in how the underlying process APIs handle it but I'm not 100% sure.